login.css

```diff
@@ -79,53 +53,65 @@ nav a{
 }
 
 
+ul li ul{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	left: 130px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+}
 
-.content_1{
+/* Set up position for content div to hold the login function */
+.content{
 	background-color: white;
+	z-index:4;
+	height:680px;
 	grid-column-start: 1;
-	grid-column-end: span 4;
-	z-index: 2;	
-	height: 100px;
-	border: 2px solid red;
+	grid-column-end: span 1;
+	
 }
 
-.section3{
-	background-color: white;
-	width: 400px;
-	height: 300px;
-	padding: 10px;
-	position: absolute;
-	top: 550px;
-	left: 35%;
-	z-index: 4;
-}
 
-.content{
+.img_2{
 	background-color: white;
 	z-index: 2;
 	height:680px;
+	grid-column-start: 2;
+	grid-column-end: span 4;
+}
+
+.img_2 img{
+	height:680px;
+	width:100%;
+}
+/* Set up the footer */
+.footer{
+	background-color: white;
 	grid-column-start: 1;
 	grid-column-end: span 4;
+	text-align: center;
+	height:35px;
+	color:black;
+	padding-top:20px;
 }
 
-.section3 h3{
-	margin-left:80px;
+
+/* Re position of text in the div */
+h3{
+	margin-left:130px;
 	margin-top:30px;
 	
 }
 
-.section3 p{
```

Comment:
01/08/2021: I have delete the section3 so the section3 text doesn't exit. So I also delete it to reduce comfusion.